bgpd/rfapi/rfapi_vty.c

```diff
@@ -5178,7 +5178,7 @@ vnc_add_vrf_prefix (struct vty *vty,
 
 DEFUN (add_vrf_prefix_rd_label_pref,
        add_vrf_prefix_rd_label_pref_cmd,
-      "add vrf NAME prefix <A.B.C.D/M|X:X::X:X/M> [{rd ASN:nn_or_IP-address|label (0-1048575)|preference (0-4294967295)}]",
+      "add vrf NAME prefix <A.B.C.D/M|X:X::X:X/M> [{<rd ASN:nn_or_IP-address|label (0-1048575)|preference (0-4294967295)>}]",
```

Comment:
* [] means 'optional'
* {} means 'zero or more of these in any order'
* <> means 'exactly one of these'

I'm not sure what the intention is here, but adding <> isn't correct. Also if the parser isn't rejecting `[{foo|bar}]` I believe that needs to be fixed, as well as `list permutations` since it shouldn't crash on any command.